cms/templates/home_page.html

```diff
@@ -70,42 +54,6 @@ <h3 class="featured-product-header-title col-8">{{ product_cards_section_title }
       </div>
     </div>
   </div>
-{% else %}
-  <div class="course-section">
-    <div class="container">
-      {% if product_cards_section_title %}
-        <h1>{{ product_cards_section_title }}</h1>
-      {% endif %}
-      <ul class="course-listing card-listing">
```

Comment:
course-listing is not used anywhere else and should be removed

Reply:
removed